### ISSUE TITLE
Perform Functional Tests

### ISSUE THREAD

**Issue by LuxMiranda** [SmallSatGasTeam/errno_3#63]

**Comment by nick-benoit14** [comment]
Do you remember what this was about?

**Comment by LuxMiranda** [comment]
Yes - we just want to ensure that auto-deployment still works after 'b' has been pressed but before 'y' has been pressed in the event that we lose communication with the payload right at that moment. It should be a quick and simple test to perform.

**Comment by nick-benoit14** [comment]
This is covered in a unit test, but we should make sure this is one of the things we do during our functional test. I'm going to change the name of this issue to 'Functional Test' meaning, that we should make sure everything acts as expected in the vacuum chamber. This is one of those tests we should do I think.
